### PR TITLE
Update GetRegionData "returns" section

### PR DESCRIPTION
**Pull request by Pavel-Cherentsov**
Remove duplicate "if the function fails, the return value is zero"

### CODE REVIEW

File: sdk-api-src/content/wingdi/nf-wingdi-getregiondata.md [hunk 1]
```diff
@@ -74,9 +74,11 @@ A pointer to a <a href="/windows/desktop/api/wingdi/ns-wingdi-rgndata">RGNDATA</
 
 ## -returns
 
-If the function succeeds and <i>dwCount</i> specifies an adequate number of bytes, the return value is always <i>dwCount</i>. If <i>dwCount</i> is too small or the function fails, the return value is 0. If <i>lpRgnData</i> is <b>NULL</b>, the return value is the required number of bytes.
+If the function succeeds and <i>dwCount</i> specifies an adequate number of bytes, the return value is always <i>dwCount</i>.
```
Comment (Pavel-Cherentsov) [inline sdk-api-src/content/wingdi/nf-wingdi-getregiondata.md]: Apparently this is incorrect.
If the function succeeds and <i>dwCount</i> specifies a sufficient number of bytes, the return value is the actual number of bytes used (less than or equal to <i>dwCount</i>).

**Comment by stevewhims** [comment]
Thanks, I'm investigating.

**Comment by stevewhims** [comment]
Still trying to get info about whether this is the right change.

**Review by stevewhims — APPROVED** [review]

**Comment by stevewhims** [comment]
Hi, @Pavel-Cherentsov, it looks like the CLA hasn't been signed yet (please see comment in the thread above). If you could sign that, then the button will appear on this end for me to complete the PR. Thanks! :)

**Comment by stevewhims** [comment]
Hi, @Pavel-Cherentsov, it looks like the CLA hasn't been signed yet (please see comment in the thread above). If you could sign that, then the button will appear on this end for me to complete the PR. Thanks! :)

**Comment by stevewhims** [comment]
Hi, @Pavel-Cherentsov, it looks like the CLA hasn't been signed yet (please see comment in the thread above). If you could sign that, then the button will appear on this end for me to complete the PR. Thanks! :)

**Pull request merged**